.travis.yml

```diff
@@ -1,16 +1,24 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.4"
-  - "3.5"
+matrix:
+  include:
+    - python: "2.6"
+      env: TOXENV=py26
+    - python: "2.7"
+      env: TOXENV=py27
+    - python: "3.4"
+      env: TOXENV=py34
+    - python: "3.5"
+      env: TOXENV=py35
 # command to install dependencies
 install:
   - "pip install -r requirements.txt"
```

Comment:
>In theory setup.py would pull in the requirements from requirements.txt

This is probably the best option (see https://github.com/vrutkovs/github-review-dashboard/blob/master/setup.py#L15), but I'm not sure if py26 supports that